recipes/conversation_view/feed_pill/feed_item_pill.vue

```diff
@@ -103,6 +103,11 @@ export default {
       default: () => true,
     },
 
+    defaultToggled: {
```

Comment:
Open for feedback for a better naming, `initialyToggled`?

Reply:
You can use `isToggled` use the prefix convention `is`, `has`, `can` when you're defining boolean values.

Reply:
Yes, my doubt with `isToggled` is that it kind of also infers the meaning that can be toggled through that, like we do with the `open` property from the `collapsible` component

Reply:
Hmm I understand, so `defaultExpanded` might work I think